centreon/www/front_src/src/AgentConfiguration/Specs/AgentConfigurations.cypress.spec.tsx

```diff
@@ -802,92 +800,92 @@ describe('Agent configurations modal', () => {
-    initialize({});
-
-    cy.contains(labelAdd).click();
-
-    cy.findByLabelText(labelAgentType).click();
-    cy.get('[data-option-index="0"]').click();
-
-    cy.findByLabelText(labelEncryptionLevel).click();
-    cy.contains(labelInsecure).click();
-
-    cy.findByLabelText(labelName).type('Insecure Agent');
-    cy.findByLabelText(labelPollers).click();
-    cy.contains('poller1').click();
-    cy.findAllByLabelText(labelPort).eq(0).clear().type('1234');
-
-    cy.findAllByLabelText(labelPublicCertificate).eq(0).should('exist');
-    cy.findByLabelText(labelCaCertificate).should('exist');
-    cy.findAllByLabelText(labelPrivateKey).should('have.length', 2);
-
-    cy.contains(labelSave).should('be.disabled');
-
-    cy.findAllByLabelText(labelPublicCertificate).eq(0).type('test.crt');
-    cy.findByLabelText(labelCaCertificate).type('ca.crt');
-    cy.findAllByLabelText(labelPrivateKey).eq(0).type('test.key');
-    cy.findAllByLabelText(labelPrivateKey).eq(1).type('test.key');
-    cy.findAllByLabelText(labelPublicCertificate).eq(1).type('test.cer');
-
-    cy.contains(labelSave).should('be.enabled');
-
-    cy.makeSnapshot();
-
-    cy.contains(labelSave).click();
-
-    cy.waitForRequest('@postAgentConfiguration').then(({ request }) => {
-      expect(request.body).to.deep.equal({
-        name: 'Insecure Agent',
-        connection_mode: 'insecure',
-        type: 'telegraf',
-        configuration: {
-          otel_private_key: 'test.key',
-          otel_ca_certificate: 'ca.crt',
-          otel_public_certificate: 'test.crt',
-          conf_certificate: 'test.cer',
-          conf_private_key: 'test.key',
-          conf_server_port: 1234
-        },
-        poller_ids: [1]
-      });
-    });
-
-    cy.contains(labelAgentConfigurationCreated).should('be.visible');
-  });
-
-  it('sends an update request with certificate fields when connection mode is insecure', () => {
-    initialize({});
-
-    cy.contains('AC 1').click();
-    cy.waitForRequest('@getAgentConfiguration');
-
-    cy.findByLabelText(labelEncryptionLevel).click();
-    cy.contains(labelInsecure).click();
-
-    cy.findByLabelText(labelName).clear().type('Insecure Agent');
-
-    cy.makeSnapshot();
-
-    cy.contains(labelSave).click();
-
-    cy.waitForRequest('@patchAgentConfiguration').then(({ request }) => {
-      expect(request.body).to.deep.equal({
-        name: 'Insecure Agent',
-        type: 'telegraf',
-        connection_mode: 'insecure',
-        configuration: {
-          otel_private_key: 'test.key',
-          otel_ca_certificate: 'test.crt',
-          otel_public_certificate: 'test.cer',
-          conf_certificate: '/sub/test.crt',
-          conf_private_key: 'test.crt',
-          conf_server_port: 9090
-        },
-        poller_ids: [1, 2]
-      });
-    });
-
-    cy.contains(labelAgentConfigurationUpdated).should('be.visible');
-  });
+  // it('send a Post request with certificate fields when connection mode is insecure', () => {
```

Comment:
Will it be re-enabled soon? If not, remove the code

Reply:
yes, it will be re-enabled soon